src/modules/TftpServer.rb

```diff
@@ -23,36 +32,45 @@ def main
 
       Yast.import "Progress"
       Yast.import "Report"
-      Yast.import "Service"
+      Yast.import "SystemdSocket"
+      Yast.import "SystemdService"
       Yast.import "Summary"
-      Yast.import "SuSEFirewall"
 
       # Any settings modified?
       # As we have only a single dialog which handles it by itself,
       # it is used only by autoinst cloning.
       @modified = false
 
       # Required packages for operation
-      @required_packages = ["xinetd", "tftp"]
+      @required_packages = [PACKAGE_NAME]
 
-      # Start tftpd via xinetd?
+      # if socket start tftp
       @start = false
 
-      # Image directory, last argument of in.tftpd
-      @directory = ""
-
-      # Other arguments to in.tftpd, ie. not including -s or /dir
-      @other_args = ""
-
+      # sysconfig model
+      @sysconfig = ::CFA::TftpSysconfig.new
+      # sysconfig values we are interested in. Allow to change it with UI.
+      # TODO when doing bigger changes use sysconfig model everywhere
+      @directory = "/src/tftpboot" # default value
```

Comment:
Did you mean ```/srv/tftproot```

Reply:
no, I use same default as in https://build.suse.de/package/view_file/SUSE:SLE-15:GA/tftp/tftp.sysconfig?expand=1
where is also tftpboot

Reply:
I mean src vs srv